a8c-files.php

```diff
@@ -71,15 +71,6 @@ private function init_jetpack_photon_filters() {
 		// This results in all images being full size (which is not ideal)
 		add_filter( 'jetpack_photon_development_mode', '__return_false', 9999 );
 
-		// The sizes metadata is not used and mostly useless on Go so let's empty it out.
```

Comment:
It has just occurred to me that we might need to keep this in place or backfill in case the srcset implementation is turned off.

I'll look into it.

Reply:
The main reason we added this in was to avoid issues on migrated sites with stale attachment data that reference hard-coded image path in the sizes meta, which would break things.

If we restore this, we'll need to remember to fix the logic in `maybe_inject_image_sizes`